news/2022-05-21-project-loved-may-2022.md

```diff
@@ -0,0 +1,321 @@
+
+*9mm* has gained quite a following over the years, and for good reason! Be sure to cast your vote for it in this wave of Project Loved.
+
+[![](/wiki/shared/news/2022-05-21-project-loved-may-2022/358342.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[UNDEAD CORPORATION - The Empress](https://osu.ppy.sh/beatmapsets/358342#mania) by [\_underjoy](https://osu.ppy.sh/users/2235750)\
+270 BPM, 4:20 | 6.58★\
+*description written by [-NoName-](https://osu.ppy.sh/users/10400730)*
+
+Next up is \_underjoy's chart of *The Empress*!
+
+If you are familiar with \_underjoy's [ranked set](https://osu.ppy.sh/beatmapsets/315435) of *The Empress scream off ver*, you will also quickly become familiar with his longer chart of *The Empress*. This chart is packed to the brim with relentless stamina-draining patterns at 270 BPM. If you ever wanted to give a longer version of the ranked set a leaderboard, now is your chance.
+
+Set this chart of *The Empress* free from the deepest nightmare that is the graveyard and cast your vote!
+
+[![](/wiki/shared/news/2022-05-21-project-loved-may-2022/1098451.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Senya - Melody Blooming in the Earth](https://osu.ppy.sh/beatmapsets/1098451#mania) by [\[Crz\]Rachel](https://osu.ppy.sh/users/8369409)\
+132 BPM, 4:32 | 5.80★, 6.27★, 6.76★, 7.26★, 7.74★\
+*description written by [RandomeLoL](https://osu.ppy.sh/users/7080063)*
+
+It took some months for them to come back into the spotlight, but they make a strong reappearance in the Loved polls: \[Crz\]Rachel is yet again offering the community one of their well-known chordjack charts, this time *Senya's Melody Blooming in the Earth*!
+
+To be frank, it has been fairly difficult to choose something from them as they have tons of charts Loved by the community. However, we believe that the rate accessibility of this chart in particular might be more enjoyed across the board! As a player, you should expect heavily anchored sections, a recurring motif in the chart. These seem to spice the experience however, as getting to control them is what's going to give you an edge.
+
+Other than that, not much else can be said. Just a gamer tip; try hitting the note. It tends to work fairly well.
+
+[![](/wiki/shared/news/2022-05-21-project-loved-may-2022/1012105.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Rish feat. Choko - Bi](https://osu.ppy.sh/beatmapsets/1012105#mania) by [Raveille](https://osu.ppy.sh/users/1388767)\
+169 BPM, 1:54 | 4.90★\
+*description written by [Paturages](https://osu.ppy.sh/users/1375479)*
+
+Contrary to popular assumption(?), this song is not about the "B" of LGBT, nor is it about the letter "B" itself, the insect known as "bee" or its dedicated movie, or even the 🅱️ emoji. There's only one way to find out whether or not Unicode emojis work in news posts, let's see how this goes.
```

Comment:
i mean what do you expect, of course it will

```suggestion
Contrary to popular assumption(?), this song is not about the "B" of LGBT, nor is it about the letter "B" itself, the insect known as "bee" or its dedicated movie, or even the 🅱️ emoji.
```

Reply:
idk either but this was captains joke so imma keep it